test/lint/lint-include-guards.sh

```diff
@@ -12,7 +12,7 @@ HEADER_ID_PREFIX="ZCASH_"
 HEADER_ID_PREFIX_UPSTREAM="BITCOIN_"
 HEADER_ID_SUFFIX="_H"
 
-REGEXP_EXCLUDE_FILES_WITH_PREFIX="src/(leveldb/|secp256k1/|univalue/)"
+REGEXP_EXCLUDE_FILES_WITH_PREFIX="src/(crypto/ctaes/|leveldb/|secp256k1/|tinyformat.h|univalue/)"
```

Comment:
Why are these excluded rather than just fixed?

Reply:
Per the commit message:

> lint: Re-exclude subtrees from lint-include-guards.sh
>
> These are external libraries, and it does not make sense to maintain an
> otherwise-meaningless diff from upstream.

Reply:
For more meaningful lint changes (like locale, or the changes we've made to `secp256k1`), sure. The include guards have zero impact on our codebase (they've been working fine as-is), so it just adds noise.